lacecore/_obj.py

```diff
@@ -0,0 +1,79 @@
+from collections import OrderedDict
+import numpy as np
+
+from ._group_map import GroupMap
+from ._mesh import Mesh
+
+ERROR_MESSAGE = "tinyobjloader library has not been installed. You will not be able to load OBJ files"
+
+try:
+    from tinyobjloader import ObjReader, ObjReaderConfig
+except Exception:  # pragma: no cover
+    ObjReader = None
+    ObjReaderConfig = None
+    import sys
+
+    print("WARNING: %s" % ERROR_MESSAGE, file=sys.stderr)
```

Comment:
Trimesh throws a bunch of warnings on import when optional dependencies aren't installed, and I think it doesn't make sense. It sort of implies that the dependencies are required. I'd suggest we remove this.

Reply:
Done